gulp/tests.js

```diff
@@ -4,26 +4,10 @@ import path from 'path'
 import gutil from 'gulp-util'
 import {paths} from './paths'
 import webdriver from 'gulp-webdriver'
-import selenium from 'selenium-standalone'
 import yargs from 'yargs'
 
 const KarmaServer = karma.Server
 
-export let seleniumServer
-
-export function startSeleniumServer(done) {
```

Comment:
@benjefferies Yeah this code was removed as newer versions of webdriver include the Selenium server so you don't have to spin one up manually anymore.